src/coverlet.core/Reporters/IReporter.cs

```diff
@@ -2,6 +2,7 @@ namespace Coverlet.Core.Reporters
 {
     public interface IReporter
     {
+        bool UseConsoleOutput { get; }
```

Comment:
We should create an enum `ReporterOutputType`:

```csharp
enum ReporterOutputType
{
    File,
    Console,
}
```

This will help us to potentially support more output types in the future

Reply:
Added `ReporterOutputType` enum to the `IReporter` file. Seemed like as good a place as any 🤷‍♂️